mappings/net/minecraft/util/math/ChunkPos.mapping

```diff
@@ -23,6 +23,14 @@ CLASS net/minecraft/class_1923 net/minecraft/util/math/ChunkPos
 		ARG 1 pos2
 	METHOD method_24022 getChebyshevDistance (Lnet/minecraft/class_1923;)I
 		ARG 1 pos
+	METHOD method_33939 getOffsetX (I)I
```

Comment:
Is this converting a local x in a chunk to a global x? I suggest `offsetX()` if that's the case.

Reply:
It's adding a value on the x and z to the block coords of the chunk's start, everything else uses `get` so I think this should too

Reply:
how about calling this get global x/z and treat inputs as local x/zs? imo global is clearer than offset

Reply:
After looking at actual usages, this name looks good.